soci/soci_index.go

```diff
@@ -40,23 +40,23 @@ import (
 )
 
 const (
-	// artifactType of index SOCI index
+	// SociIndexArtifactType is the artifactType of index SOCI index
 	SociIndexArtifactType = "application/vnd.amazon.soci.index.v1+json"
-	// mediaType of ztoc
+	// SociLayerMediaType is the mediaType of ztoc
 	SociLayerMediaType = "application/octet-stream"
-	// index annotation for image layer media type
+	// IndexAnnotationImageLayerMediaType is the index annotation for image layer media type
 	IndexAnnotationImageLayerMediaType = "com.amazon.soci.image-layer-mediaType"
-	// index annotation for image layer digest
+	// IndexAnnotationImageLayerDigest is the index annotation for image layer digest
 	IndexAnnotationImageLayerDigest = "com.amazon.soci.image-layer-digest"
-	// index annotation for build tool identifier
+	// IndexAnnotationBuildToolIdentifier is the index annotation for build tool identifier
 	IndexAnnotationBuildToolIdentifier = "com.amazon.soci.build-tool-identifier"
-	// media type for OCI Artifact manifest
+	// OCIArtifactManifestMediaType is the media type for OCI Artifact manifest
 	OCIArtifactManifestMediaType = "application/vnd.oci.artifact.manifest.v1+json"
-	// default span size (4MiB)
+	// defaultSpanSize (4MiB)
 	defaultSpanSize = int64(1 << 22)
-	// min layer size (10MiB)
+	// minLayerSize to build a ztoc for a layer (10MiB)
```

Comment:
We should have better explanation of what "min layer" means.

Reply:
A better place to explain this is the cli flag [`Usage`](https://github.com/awslabs/soci-snapshotter/blob/1e87baff6ad9fec046c39e3ed7c4245b40efea79/cmd/soci/commands/create.go#L55) which is user-facing and [`WithMinLayerSize `](https://github.com/awslabs/soci-snapshotter/blob/1e87baff6ad9fec046c39e3ed7c4245b40efea79/soci/soci_index.go#L148) which is exported.

Do you think the cli usage is clear enough? (`The minimum layer size in bytes to build zTOC for. Default is 10 MiB.`) Or we can change to `The minimum layer size in bytes to build zTOC for. Default is 10 MiB. Layers smaller than this won't have zTOC (and not lazy pulled).`